projects/1.19/assets/more-crossbows-cj/morecrossbows/lang/zh_cn.json

```
@@ -0,0 +1,40 @@
+{
+    "item.morecrossbows.copper_crossbow": "铜弩",
+    "item.morecrossbows.iron_crossbow": "铁弩",
+    "item.morecrossbows.golden_crossbow": "金弩",
+    "item.morecrossbows.emerald_crossbow": "绿宝石弩",
+    "item.morecrossbows.diamond_crossbow": "钻石弩",
+    "item.morecrossbows.netherite_crossbow": "下界合金弩",
+    "item.morecrossbows.gilded_netherite_crossbow": "镀金下界合金弩",
+    "item.morecrossbows.enderite_crossbow": "末影合金弩",
+    "item.morecrossbows.gilded_enderite_crossbow": "镀金末影合金弩",
+    
+    "upgrade.morecrossbows.crossbow_upgrade": "升级弩",
+	"item.morecrossbows.smithing_template.crossbow_upgrade.additions_slot_description": "把下一级的方块放在这里。",

```

| **审查级别**    |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思  |

原文：**Put a Block of the next tier here.**
```suggestion
	"item.morecrossbows.smithing_template.crossbow_upgrade.additions_slot_description": "把下一级材料的块放在这里。",
```

---

修改于 8d8c0c5